docs/developer-docs/multi-chain/ethereum/evm-rpc/costs.mdx

```diff
@@ -37,7 +37,7 @@ Here is an approximate cost breakdown in USD for an RPC request on the Ethereum
 
 - Total cost: ~ $0.0021 (assuming consensus between 3 RPC providers)
 
-  - Cost per RPC provider: ~ $0.000686 (assuming 1kb request and 1kb response)
+  - Cost per RPC provider: ~ $0.000686 (assuming 1kB request and 1kB response)
```

Comment:
Shouldn't it be 1KiB?

Reply:
The calculation was based on 1000 request / response bytes, which should be "kB" if I remember correctly (where KiB is 1024 bytes)?

Reply:
Got it, the current version is correct then.